scripts/test_binary.py

```diff
@@ -65,19 +65,20 @@
 
     # binary zero check
     # Checks 10 random stations for any occurances of 0 in the output (aka results have not been written)
+    # Removes leading 0s from the test as there may be some time at the start before the waveform starts.
     for stat_name in np.random.choice(
         bin.stations.name, replace=False, size=min(10, bin.stations.shape[0])
     ):
 
         acc = bin.acc(stat_name)
-
-        if np.any(acc == 0):
-            if args.verbose:
-                print(
-                    f"The velocities for station {stat_name} contains zero/s, please investigate. This "
-                    f"is most likely due to crashes during HF or BB resulting in no written output."
-                )
-            #  sys.exit(1)  # Commenting this line out until further compatibility investigations - Jason: 20-10-30
+        for comp in acc.T:
```

Comment:
will it not be better to use transpose()? Last time I had issue with this while using newer version of numpy and pandas.

Reply:
Apparently they're the same operation? https://stackoverflow.com/questions/4068177/whats-the-difference-of-numpy-ndarray-t-and-numpy-ndarray-transpose-when-self

Reply:
ah, yes, you are correct. I just double checked my commit, it was the returned datatype that had issue, not the method.